playbooks/openshift-hosted/private/create_persistent_volumes.yml

```diff
@@ -10,6 +10,7 @@
     persistentlocalstorage_project: "{{ openshift_persistentlocalstorage_project | default('local-storage') }}"
     persistentlocalstorage_classes: "{{ openshift_persistentlocalstorage_classes | default([]) }}"
     persistentlocalstorage_path: "{{ openshift_persistentlocalstorage_path | default('/mnt/local-storage') }}"
+    persistentlocalstorage_image: "{{ persistentlocalstorage_provisioner_image | default('quay.io/external_storage/local-volume-provisioner:v2.2.0') }}"
```

Comment:
Any reason we're passing variables in this way?  Could probably just be refactored into role defaults?

Reply:
Mostly to shorten it. Role defaults sound like a better idea